pkg/services/serviceaccounts/database/database.go

```diff
@@ -294,8 +294,6 @@ func (s *ServiceAccountsStoreImpl) UpdateServiceAccount(ctx context.Context,
 }
 
 func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(ctx context.Context, query *models.SearchOrgUsersQuery) ([]*serviceaccounts.ServiceAccountDTO, error) {
-	query.IsServiceAccount = true
```

Comment:
question: now that service account and user DB stores have been separated, do we still use `IsServiceAccount` of this query? If not, maybe it can be removed altogether.

Reply:
there might still be some instances of this in the user store which aren't handled yet

Reply:
From what I could see, I think there's one place where it's used (user listing), so we could just filter for `is_service_account = false` there. But up to you, I'm also fine with this being cleaned up later :)

Reply:
there's some places in users that are not filtering SAs correctly which need to be added in https://github.com/grafana/grafana-enterprise/issues/2849